src/libmongoc/src/mongoc/mongoc-bulk-operation.c

```diff
@@ -394,15 +401,23 @@ mongoc_bulk_operation_insert_with_opts (mongoc_bulk_operation_t *bulk,
       }
    }
 
+   if (bulk->comment.value_type != BSON_TYPE_EOD) {
+      bson_append_value (&cmd_opts, "comment", 7, &bulk->comment);
+   }
+
+   /* TODO: mongoc_bulk_insert_opts_t specifies allow_extra=False, so there was
+    * no reason to previously pass &insert_opts.extra as cmd_opts */
```

Comment:
This TODO exists for the benefit of code review. Please confirm that my understanding is correct and I'll remove this accordingly.